src/main/java/frc/robot/commands/autos/OneFiveBumpReturn.java

```diff
@@ -0,0 +1,109 @@
+package frc.robot.commands.autos;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Cal;
+import frc.robot.commands.IntakeSequence;
+import frc.robot.commands.autos.components.DriveDistance;
+import frc.robot.commands.autos.components.ScoreThisGamePiece;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.IntakeLimelight;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.Lights;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.utils.ScoringLocationUtil;
+
+/**
+ * Assuming the robot is starting from cone scoring position nearest the loading zone, this auto
+ * component drives to the nearest midfield game object, picks it up, and returns to starting
+ * position
+ */
+public class OneFiveBumpReturn extends SequentialCommandGroup {
+  private static final double DISTANCE_AT_CONE_METERS = 5.9;
+  private static final double NORM_SPEED_INTAKING = 0.3;
+  private double startXMeters = 0;
```

Comment:
I don't think we need this variable anymore for either of these autos, same for `DISTANCE_AT_CONE_METERS `